<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestSetup</class>
 <widget class="QWidget" name="TestSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="4">
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>The person who performs the test </string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_19">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Check if displays have been calibrated</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>The person or company that performs the test</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Test setup</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="TestReason_lineEdit">
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_12">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>YYYY-MM-DD</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QDateEdit" name="TestDate">
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_17">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Description of environmental conditions</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>The date when the workstation test is performed</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="Tester_lineEdit">
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Test date</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QTextEdit" name="PerformedBy_textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Tester</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string>Performed by</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QTextEdit" name="TestSetup_textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_18">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Displays calibrated</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Test reason</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>The reason why the workstation have to been tested</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QComboBox" name="DisplayCalibrated_comboBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>--</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>no</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_25">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>QA software</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Name / Version</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="Vendor_lineEdit">
       <property name="minimumSize">
        <size>
         <width>321</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>321</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="QaSoftwareName_lineEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_29">
       <property name="minimumSize">
        <size>
         <width>515</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>515</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Name and version of Qa test software</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_27">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Vendor of QA test software</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_26">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>220</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Vendor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="QaSoftwareVersion_lineEdit">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
